src/content/docs/infrastructure/host-integrations/host-integrations-list/aerospike-integration.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Aerospike platform integration
+tags:
+    - New Relic integrations
+    - Aerospike
+metaDescription: Use New Relic's infrastructure monitoring to get a dashboard with metrics from your Aerospike platform. 
+
+---
+
+import infrastructureAerospikeDashboard from 'images/infrastructure_screenshot-full_aerospike-dashboard.png'
+
+Our Aerospike integration monitors the performance of your Aerospike data platform, helping you diagnose issues in your application and optimize your code. Our Aerospike integration gives you a pre-built dashboard with your most important Aerospike app metrics.
+
+<img
+    title="aerospike dashboard"
+    alt="A screenshot depicting an Aerospike pre-built dashboard"
+    src={infrastructureAerospikeDashboard}
+/>
+
+<figcaption>
+After setting up our Aerospike integration, we give you a dashboard for your Aerospike platform metrics. 
+</figcaption>
+
+## Install [#install]
+
+### Step 1 (optional): Install the infrastructure agent [#infra-install]
+
+Installing the infrastructure agent is optional, but doing this will get you important metrics about the host where your app is running. 
```

Comment:
Installing the infrastructure agent will get you important metrics about the host where your app is running.